middlewares/members.js

```diff
@@ -63,6 +64,18 @@ exports.updateUser = async (req, res) => {
     }
 
     await req.currentUser.update(req.body, { fields: constants.FIELDS_TO_UPDATE.USER.UPDATE });
+
+    // TODO: update first/late name to GSuite account (if there is an account attached)
```

Comment:
I mention it here, but it goes for the whole file; not every user will have a GSuite account attached (mostly older accounts) so we also need to check that we only do things if an account is attached